frontend/src/components/Modal/index.jsx

```diff
@@ -0,0 +1,134 @@
+//               <img src={Exit} alt="close" id="close-button" />
+//             </Button>
+//           </div>
+//           <form onSubmit={this.handleSubmit} className="add-edit-resource-form">
+//             <label>
+//               Resource Name
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Resource Location
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Resource Tags
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//             <label>
+//               Description
+//               <input type="text" onChange={this.handleChange} />
+//             </label>
+//           </form>
+//           <div className="lah-modal-footer">
+//             <Button
+//               color="info"
+//               onClick={this.props.toggleModal}
+//               id="save-button"
+//             >
+//               Save
+//             </Button>
+//           </div>
+//         </section>
+//       </div>
+//     );
+//   }
+// }
+
+// export default Modal;
+
+import React, { Component } from "react";
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
+import "./styles.scss";
+
+class LAHModal extends Component {
+  render() {
+    return (
+      <div>
+        <Modal fade={false} isOpen={this.props.showModal}>
+          <ModalHeader>{this.props.modalName}</ModalHeader>
+          <ModalBody
+            style={{
+              "max-height": "calc(100vh - 210px)",
+              "overflow-y": "auto"
+            }}
+          >
+            <form
+              onSubmit={this.handleSubmit}
+              className="add-edit-resource-form"
+            >
+              <label>
+                Resource Name
+                <input type="text" onChange={this.handleChange} />
```

Comment:
I figure this is part of the PR being WIP but just wanted to make sure these handlers like `handleChange` map to a function.